### ISSUE TITLE
Working example?

### ISSUE THREAD

**Issue by Ramblurr** [fbrbovic/kubernetes-reverseproxy#5]
This looks interesting, however the minimal docs leaves me with questions.

How does nginx map from a hostname to a minion_node_ip:port?

I'd love to see a working example!

**Comment by nhumrich** [comment]
+1

**Comment by SleepyBrett** [comment]
Yes this seems like a great container, but the docs make it hard to get running properly.

**Comment by jonesrussell** [comment]
It's been awhile since I've been doing R&D and got this working in my local Kubernetes cluster.

This is how I start the kubernetes-reverseproxy container:

```
docker run -d -e CONFD_ETCD_NODE=172.17.42.1:4001 -t -p 80:80 darkgaro/kubernetes-reverseproxy:latest
```

confd will be running in the kubernetes-reverseproxy container and it will connect to Kubernetes instance of etcd for metadata.

In my Kubernetes repository I have modified 'hack/lib/etcd.sh' from:

```
etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null &
```

to

```
etcd -data-dir ${ETCD_DIR} -addr ${testhost}:${port} --bind-addr ${host}:${port} >/dev/null 2>/dev/null &
```

I start Kubernetes like this:

```
ETCD_HOST=0.0.0.0 ETCD_PUBLIC_HOST=172.17.42.1 hack/local-up-cluster.sh
```

That's what worked for me and I don't recall if I did anything else. I have it running now and I'm happy to answer anymore questions.
